Add unbonding page to democracy (#1380)
w3f/polkadot-wiki: docs/maintain-guides-democracy.md

Like [vesting](https://wiki.polkadot.network/docs/en/learn-DOT#lazy-vesting), the tokens that are
locked in democracy are unlocked lazily. This means that you, the user, must explicitly call an
unlock extrinsic to make your funds available again after the lock expires. Unbonding is another

> term you heard a lot in Polkadot, where you would like to withdraw your DOT that used in Staking. To

Reviewer: ```suggestion
term you hear a lot in Polkadot, it means withdrawing your DOT that was used in staking. To
```
